projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -97,6 +99,29 @@ public static function register_core_options() {
 		( new Jetpack_Social_Settings\Dismissed_Notices() )->register();
 	}
 
+	/**
+	 * Things to do on admin_init.
+	 */
+	public static function admin_init_actions() {
+		( new Jetpack_Social_Settings\Dismissed_Notices() )->register();
```

Comment:
This will already be run as part of register_core_options

Reply:
Removed in e6e84d69d5fb97bd7dc2f9dd8dd89b29cc4b85f6.

Thanks